rows/sile.pldb

```diff
@@ -3,7 +3,10 @@ appeared 2012
 type textMarkup
 description SILE is a typesetting system. Its job is to produce beautiful printed documents from raw content.
 website https://sile-typesetter.org
+reference https://tug.org/TUGboat/tb38-1/tb118cozens.pdf
 reference https://news.ycombinator.com/item?id=22393099
+country Australia
+originCommunity Worldview Center for Intercultural Studies
```

Comment:
Where did you find this information in the context of SILE Markup?

Reply:
@celtic-coder : The last page in the pdf at https://tug.org/TUGboat/tb38-1/tb118cozens.pdf .
The author self identifies their affiliation. 
The creators name is Simon, but that field it not filled up yet.

Reply:
Hi Hari (@SRS-WRKS), that's quite a bit of detective work! Well done! 👍 

It is certainly an interesting background on the steps that were required to develop the SILE Typesetter. I learned quite a lot just from this article.

Good work, Hari!

Kind Regards,
Liam

Reply:
@celtic-coder : Thank you for the kind words. Have a nice day :)